Makefile.psx.mk

```diff
@@ -36,6 +38,12 @@ $(BUILD_DIR)/tt_%.ld: $(CONFIG_DIR)/splat.$(VERSION).tt_%.yaml $(PSX_BASE_SYMS)
 $(BUILD_DIR)/weapon.ld: $(CONFIG_DIR)/splat.$(VERSION).weapon.yaml $(PSX_BASE_SYMS) $(CONFIG_DIR)/symbols.$(VERSION).weapon.txt $(CONFIG_DIR)/symbols.$(VERSION).weapon.txt.in $(SPLAT_APP)
 	$(SPLAT) $<
 
+extract_assets: $(SOTNASSETS)
+	$(SOTNASSETS) extract -stage_ovl disks/$(VERSION)/ST/NZ0/NZ0.BIN -o assets/st/nz0_test
+build_assets: $(SOTNASSETS)
+	$(SOTNASSETS) build -file assets/st/nz0_test/rooms.json -kind rooms -o src/st/nz0/
+	$(SOTNASSETS) build -file assets/st/nz0_test/layers.json -kind layers -o src/st/nz0/
```

Comment:
Can you add these JSON configs to the draft?

Reply:
Those JSONs are getting generated by `make extract_assets`!

Reply:
I realized that after I made the comment. Sorry.